Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -148,6 +160,10 @@ int main(void)
     /* USER CODE BEGIN RTOS_THREADS */
     //TODO: Create threads and thread attributes
     /* add threads, ... */
+    const osThreadAttr_t blueTask_Attr = {.name = "blueTask", .priority = (osPriority) osPriorityNormal, .stack_size = 128};
+    const osThreadAttr_t greenTask_Attr = {.name = "greenTask", .priority = (osPriority) osPriorityNormal, .stack_size = 128};
+    blueThread = osThreadNew((osThreadFunc_t)blueLedToggleTask, &CANMutexAttr, &blueTask_Attr);
+    greenThread = osThreadNew((osThreadFunc_t)greenLedToggleTask, &CANMutexAttr, &greenTask_Attr);
```

Comment:
You should be passing the address of CANMutexId instead of the address of CANMutexAttr for both threads